myjsonrpc.pm

```diff
@@ -126,7 +130,6 @@ sub read_json {
 
         my $qbuffer;
         my $bytes = sysread($socket, $qbuffer, READ_BUFFER);
-        #bmwqemu::diag("sysread $qbuffer");
```

Comment:
Same as in my PR: https://github.com/os-autoinst/os-autoinst/pull/1268

Maybe Git can handle this without manual conflict resolution.

Reply:
we will see :)